tools/tsp-client/src/commands.ts

```diff
@@ -478,36 +477,43 @@ export async function generateConfigFilesCommand(argv: any) {
   if (Object.keys(overrideJson).length > 0) {
     emitterPackageJson["overrides"] = overrideJson;
   }
+  const relativeEmitterPath = argv["emitter-package-json-path"] ?? joinPaths("eng", "emitter-package.json");
+  const absoluteEmitterPath = joinPaths(repoRoot, relativeEmitterPath);
```

Comment:
I wouldn't assume that `--emitter-package-json-path` is relative to the repo root.

```suggestion
  const relativeEmitterPath = argv["emitter-package-json-path"];
  const defaultEmitterPath =  joinPaths(repoRoot, "eng", "emitter-package.json");
  const absoluteEmitterPath = relativeEmitterPath ? resolve(relativeEmitterPath) : defaultEmitterPath;
```